src/read/coff/section.rs

```
@@ -470,8 +470,25 @@ impl pe::ImageSectionHeader {
         &self,
         data: R,
     ) -> read::Result<&'data [pe::ImageRelocation]> {
-        let pointer = self.pointer_to_relocations.get(LE).into();
-        let number = self.number_of_relocations.get(LE).into();
+        let mut pointer = self.pointer_to_relocations.get(LE).into();
+        let mut number: usize = self.number_of_relocations.get(LE).into();
+        if number == core::u16::MAX.into()
+            && self.characteristics.get(LE) & pe::IMAGE_SCN_LNK_NRELOC_OVFL != 0
+        {
+            // Extended relocations. Read first relocation (which contains extended count) & adjust
+            // relocations pointer.
+            let extended_relocation_info: &pe::ImageRelocation = data
+                .read_at(pointer)
+                .read_error("Invalid COFF relocation offset or number")
+                .expect("Invalid COFF relocation offset or number");
```

This should return the error.